schema/types/ackind_test.go

```diff
@@ -43,3 +44,38 @@ func TestACKindMarshalGood(t *testing.T) {
 		}
 	}
 }
+
+func TestACKindUnmarshalBad(t *testing.T) {
+	tests := []string{
+		"garbage",
+		"ImageManifest",
```

Comment:
This is a subtlety which I should probably call out; this is not actually valid JSON, because it is not enclosed by quotes